dl-validator-data/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "dl-validator-data"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anyhow = "1.0"
```

Comment:
Since these crates are part of a workspace, maybe it makes sense to put the dependency versions at the workspace level and only `DEPENDENCY.workspace = true` in the individual crate Cargo.toml files.

Reply:
Addressed in 3e14163.